chat-plugins/scavengers.js

```diff
@@ -574,13 +574,13 @@ class ScavengerHunt extends Rooms.RoomGame {
 		if (questionArray.length % 2 === 1) return {err: "Your final question is missing an answer"};
 		if (questionArray.length < 6) return {err: "You must have at least 3 hints and answers"};
 
-		for (let i = 0; i < questionArray.length; i++) {
+		for (let [i, question] of questionArray.entries()) {
 			if (i % 2) {
-				questionArray[i] = questionArray[i].split(';').map(p => p.trim());
```

Comment:
I don't know this code so I'm wildly guessing that it depended on mutating the original array.

Reply:
Yep, it did!